### ISSUE TITLE
46900078: Voice Over reads aria-hidden elements when cursoring through webpage

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20856]
#### Description

When reading a webpage (With Safari and VoiceOver) that has elements tagged with aria-hidden="true", when you cursor up/down through the webpage those elements are spoken to the user which they should not be as they should be hidden to assistive technology.

However what does work is if you ask to read from top of document to Voice Over  cursor (control+option B)  those elements are skipped as one would expect.

Using Chrome and VoiceOver works as one would expect with all hidden elements being skipped.

Simple HTML page which demonstrates this problem.
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
    <head>
        <title>Test Aria Hidden</title>
        <meta charset="UTF-8"/>
    </head>
    <body>
        <h1>Test for Aria Hidden</h1>
        <p>Some initial text</p>
        <p aria-hidden="true">Hidden text to assistive technology.</p>
        <p>Should be next spoken text.</p>
        <p>Rest of content goes here</p>
     </body>
</html>

-
Product Version: 10.14
Created: 2018-12-21T15:35:18.279618
Originated: 2018-12-21T00:00:00
Open Radar Link: http://www.openradar.me/46900078
